test-graphql/fetch-mock-browser.js

```diff
@@ -0,0 +1 @@
+export {default as fetchMock} from 'fetch-mock/es5/client';
```

Comment:
Why not `export default from '...'`? What was the thinking around it being an explicitly named export?